keras_nlp/models/bert_test.py

```diff
@@ -45,6 +45,24 @@ def test_valid_call_bert(self):
         }
         model(input_data)
 
+    def test_variable_sequence_length_call_bert(self):
+        model = bert.Bert(
+            vocabulary_size=30522,
+            num_layers=12,
+            num_heads=12,
+            hidden_dim=768,
+            intermediate_dim=3072,
+            max_sequence_length=100,
+            name="encoder",
+        )
+        for seq_length in (25, 50, 100):
```

Comment:
Do we need to test `100`? That is covered in above tests

Reply:
sg! updated